### DISCUSSION TITLE
Reloading with sample data selected changes my layout

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#544] [Q&A]
Tested with web build.

Steps:
- Load sample dataset, which creates a new layout
- Open layouts sidebar, delete the new layout, select a different layout
- Reload the page

Expected:
- The most recently selected layout is used

Actual:
- The demo dataset is loaded again and the app adds & switches to a new copy of the demo layout, overwriting my most recent layout selection (because of the `?ds=sample-nuscenes` in the url)

**Comment by jhurliman** [comment]
<img width="261" alt="Screen Shot 2022-01-08 at 4 13 13 PM" src="https://user-images.githubusercontent.com/195374/148664726-472a4f92-27c8-483b-bc16-fe7982698a2f.png">
